content/departments/product-engineering/product/design/workshops/async-design.md

```diff
@@ -0,0 +1,45 @@
+# Async design workshop
+
+If you want to avoid sync meetings and want to have a design workshop activity that can involve a large group of people you can consider an async design workshop.
+
+The activities you do on a given day are flexible with this approach, so you can do problem validation, solution validation, brainstorming, and more; the important part is making it a multi-day activity with offsetting days for content creation and evaluation/voting, with a date boundary in between.
+
+Benefits:
+
+- Supports async work
+- More flexibility when people are in lots of time zones
+- Can support quite large groups
+- Self-documenting
+
+Drawbacks:
+
+- Plays out slower than if everyone was in a live session together
+- Not as engaging for people who are inspired by real-time communication
+- Requires lots of written communication
+
+## Offsetting days
+
+The first thing you should do when planning the workshop is identify the list of activities you want to complete. You may do some problem exploration, followed by some customer research. A way you might structure that to use offsetting days is something like this:
+
+- Monday: Problem exploration
```

Comment:
I like your summary of each day's activities [in this slack thread](https://sourcegraph.slack.com/archives/C02E3R101L3/p1644560579967029?thread_ts=1644560537.679009&cid=C02E3R101L3). Consider adding them here!